rts/motoko-rts/Cargo.toml

```diff
@@ -44,4 +44,3 @@ panic = "abort"
 [profile.release]
 panic = "abort"
 codegen-units = 1 # makes debugging easier
-opt-level = "s" # optimize for size
```

Comment:
Why this change?

Reply:
Okay, explained in https://github.com/dfinity/motoko/pull/4677#issuecomment-2402465457